frontend/src/Sidebar.res

```diff
@@ -1,31 +1,96 @@
 open! Prelude
 open Components
 
-let linkForPull = (repoId, (pullNumber, _)) => {
-  AppRouter.RepoPull({repoId: repoId, pullNumber: pullNumber})->AppRouter.path
-}
-
 let pullToString = ((pullNumber, branch)) =>
   switch branch {
   | Some(branch) => "#" ++ Belt.Int.toString(pullNumber) ++ " - " ++ branch
   | None => "#" ++ Belt.Int.toString(pullNumber)
   }
 
-module GetRepoPulls = %graphql(`
+module SidebarMenuData = %graphql(`
 query ($repoId: String!) {
-  pullNumbers: benchmarks(distinct_on: [pull_number], where: {_and: [{repo_id: {_eq: $repoId}}, {pull_number: {_is_null: false}}]}, order_by: [{pull_number: desc}]) {
+  pullsMenuData: benchmarks(distinct_on: [pull_number], where: {_and: [{repo_id: {_eq: $repoId}}, {pull_number: {_is_null: false}}]}, order_by: [{pull_number: desc}]) {
     pull_number
     branch
   }  
+  benchmarksMenuData: benchmarks(distinct_on: [benchmark_name], where: {repo_id: {_eq: $repoId}}, order_by: [{benchmark_name: asc_nulls_first}]) {
+    benchmark_name
+  }
 }
 `)
 
 module PullsMenu = {
   @react.component
-  let make = (~repoId, ~selectedPull=?) => {
+  let make = (
+    ~repoId,
+    ~data: array<SidebarMenuData.t_pullsMenuData>,
```

Comment:
I generally find `data` a confusing name unless it's the "dominant" data. For example, the data for the graph or the main dataset.

In this case, "data" for the pulls menu data I think is confusing. Specially because we already use `data` for the graphs.

Could you pick a more descriptive name? "pullsData", "pullsMenuItems"?

Reply:
That makes sense indeed 👍  I did the same for the `benchmarksMenu` to stay consistent: https://github.com/ocurrent/current-bench/pull/69/commits/3b665cae56425a0bddcbc2ad2d546a6c3ced420e